bgflow/distribution/energy/multi_double_well_potential.py

```diff
@@ -0,0 +1,47 @@
+from .base import Energy
+from bgflow.utils import distance_vectors, distances_from_vectors
+import torch
+
+
+__all__ = ["MultiDoubleWellPotential"]
+
+
+class MultiDoubleWellPotential(Energy):
+    
+    def __init__(self, dim, n_particles, a, b, c, offset):
+        super().__init__(dim)
```

Comment:
Since this is an energy that operates on multidimensional events, you should register its dimension with super().__init__(n_particles, spatial_dim)